TLM/TLM/Manager/Impl/VehicleBehaviorManager.cs

```diff
@@ -1206,7 +1208,8 @@ bool citizenDebug
             bool hasActiveTimedSimulation = (Options.timedLightsEnabled &&
                                              TrafficLightSimulationManager
                                                  .Instance.HasActiveTimedSimulation(targetNodeId));
-            bool hasTrafficLightFlag = (targetNode.m_flags & NetNode.Flags.TrafficLights) != NetNode.Flags.None;
+            NetNode.FlagsLong targetNodeFlagsLong = targetNode.flags;
```

Comment:
what was wrong with prev code?

Reply:
Node flags type has changed, I'm sure you've noticed. Functionally no change, at least for values we use😉

Reply:
the old code still works though. you don't need to deal with `NetNode.Flags2` or `NetNode.FlagsLong` if `NetNode.Flags` is all you need. is faster too.

just an observation. I am happy either-way.